bc_gym_planning_env/envs/synth_turn_env.py

```diff
@@ -213,7 +213,7 @@ def get_robot(self):
         Get the robot.
         :return IRobot: robot that is part of this environment
         """
-        return self._robot
+        return self._env.get_robot()
```

Comment:
we can't use self._robot to store the instance here, because self._env will be renewed in reset(), and it will create a new robot instance

Reply:
Agreed! Good find